python/ray/util/sgd/v2/constants.py

```diff
@@ -1,3 +1,7 @@
-from ray.util.sgd.v2.backends.torch import TorchConfig
+TIME_THIS_ITER_S = "_time_this_iter_s"
 
-BACKEND_NAME_TO_CONFIG_CLS = {"torch": TorchConfig}
+IS_FINAL_OUTPUT = "_sgd_is_final_output"
```

Comment:
Is this still needed for something? Doesn't look like it's being referenced anywhere.

Reply:
Good catch. This is a remnant of a previous implementation. Removed.